assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -257,7 +261,7 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
 
   const mainYAxis = d3.axisLeft(mainYScale)
     .tickSize(0)
-    .tickFormat(d => d.split('|')[1])
+    .tickFormat(d => truncate(d.split('|')[1]))
```

Comment:
Did you try this with `text-overflow: ellipsis` ? 
https://www.w3schools.com/cssref/css3_pr_text-overflow.asp

Reply:
Yeah I tried to modify the css but it did not work.